systems/src/apu/air_intake_flap.rs

```diff
@@ -0,0 +1,211 @@
+use super::AirIntakeFlapController;
+use crate::{shared::random_number, simulator::UpdateContext};
+use std::time::Duration;
+use uom::si::{f64::*, ratio::percent};
+
+#[derive(Debug)]
+pub struct AirIntakeFlap {
+    state: Ratio,
+    delay: Duration,
+}
+impl AirIntakeFlap {
+    const MINIMUM_TRAVEL_TIME_SECS: u8 = 3;
+
+    pub fn new() -> AirIntakeFlap {
+        let delay = Duration::from_secs(
+            (AirIntakeFlap::MINIMUM_TRAVEL_TIME_SECS + (random_number() % 13)) as u64,
```

Comment:
Wow, it varies between 3 and 15 seconds? That seems like a very wide spread. Is that randomness uniform-ish in reality? Otherwise you could use a square function or something to make it more likely to be an average value.

Reply:
3 to 15 seconds was deemed okay by Komp. I have however asked him to again verify this is reasonable as I agree with you the range does feel a bit excessive.

Reply:
For now I've changed this to "between 6 to 12 seconds". We could further make this more accurate with a square function as you described.

Komp now states:
> Rough guess: 6-12, but with a tendency to the lower end if that's possible